src/Symfony/Component/Config/Definition/Dumper/YamlReferenceDumper.php

```diff
@@ -185,7 +185,7 @@ private function writeLine($text, $indent = 0)
         $this->reference .= sprintf($format, $text)."\n";
     }
 
-    private function writeArray(array $array, $depth)
+    private function writeArray(array $array, $depth, $commented = false)
```

Comment:
Just asking: is `$commented` the best posible name for this argument? It looks like a boolean param called `isCommented`. Wouldn't `$addComments`, `$includeComments`, $maintainComments`, etc. convey a more precise meaning?